### PR TITLE
HERITAGE-123 Update 404 template to match design

### PR DESCRIPTION
**Pull request by garethmead**
Ticket URL: [HERITAGE-123](https://national-archives.atlassian.net/browse/HERITAGE-123)

## About these changes

Update 404 template to match design

## How to check these changes

404 page can be viewed at `/404`

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [ ] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

[HERITAGE-123]: https://national-archives.atlassian.net/browse/HERITAGE-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/404.html [hunk 1]
```diff
@@ -47,26 +49,34 @@
...
 
-            <div class="container mt-4 pb-6">
-                <div class="row">
-                    <div class="col-12">
-                        <p>Sorry, this page could not be found.</p>
+                            <p>If you typed the web address, check it is correct.</p>
+
+                            <p>If you pasted the web address, check you copied the entire address.</p>
+
+                            {# @TODO: url to search will need updating when connected to Rosetta API #}
```
Comment (TNA-Allan) [inline templates/404.html:65]: @garethmead Thanks for putting this, will update accordingly.

**Review by TNA-Allan — APPROVED** [review]
Looks good. Thanks @garethmead

File: templates/404.html [hunk 2]
```diff
@@ -47,26 +49,34 @@
         <main id="maincontent">
             <div class="generic-intro">
                 <div class="container">
```
Comment (ahosgood) [inline templates/404.html:51]: Can/should this be a `tna-container` with a `tna-column tna-column--full` column in it? Is the layout as expected?

Comment (garethmead) [reply templates/404.html]: It was providing the expected layout, but I've updated it to use the `tna-container` for consistency with the rest of the project. I'll make the same changes to the 500 template

For my own reference. Is the bootstrap container and grid a legacy tool that's being phased out?

Comment (ahosgood) [reply templates/404.html]: Yes it is 👍🏼

File: templates/404.html [hunk 3]
```diff
@@ -46,27 +48,32 @@
...
-                    <div class="row">
-                        <div class="col-12">
-                            <h1 class="tna-heading-xl generic-intro__title">
-                                Page not found
-                            </h1>
-                        </div>
-                    </div>
-                </div>
-            </div>
+                <div class="tna-container">
```
Comment (ahosgood) [inline templates/404.html:51]:
```suggestion
                <div class="tna-container tna-container--centred">
```

File: templates/404.html [hunk 4]
```diff
@@ -46,27 +48,32 @@
...
+                    <div class="tna-column tna-column--full tna-column--width-1-2 tna-column--full-small tna-column--full-tiny tna-!--margin-vertical-l mx-auto">
```
Comment (ahosgood) [inline templates/404.html:52]:
```suggestion
                    <div class="tna-column tna-column--full tna-column--width-1-2 tna-column--full-small tna-column--full-tiny tna-!--margin-vertical-l">
```

File: templates/404.html (hunk 3 above)
Comment (garethmead) [reply templates/404.html]: Useful to know, thanks

File: templates/404.html [hunk 5]
```diff
@@ -46,27 +48,32 @@
...
-                        <div class="col-12">
-                            <h1 class="tna-heading-xl generic-intro__title">
-                                Page not found
-                            </h1>
-                        </div>
-                    </div>
-                </div>
-            </div>
+                <div class="tna-container tna-container--centred">
+                    <div class="tna-column tna-column--full tna-column--width-1-2 tna-column--full-small tna-column--full-tiny tna-!--margin-vertical-l">
```
Comment (ahosgood) [inline templates/404.html:52]: Is there meant to be both a `tna-column--full` and a `tna-column--width-1-2`?

Comment (garethmead) [reply templates/404.html]: Well spotted 👍 Now fixed

**Review by ahosgood — APPROVED** [review]

**Pull request merged**